pkg/volume/fc/attacher.go

```diff
@@ -160,7 +161,7 @@ func (detacher *fcDetacher) UnmountDevice(deviceMountPath string) error {
 	return nil
 }
 
-func volumeSpecToMounter(spec *volume.Spec, host volume.VolumeHost) (*fcDiskMounter, error) {
+func volumeSpecToMounter(spec *volume.Spec, host volume.VolumeHost, pod *v1.Pod) (*fcDiskMounter, error) {
```

Comment:
`pod` argument is not used. Can it be removed?

Reply:
Done. pod is removed from argument.